pkg/meta/parse.go

```diff
@@ -44,13 +45,20 @@ func ParseStorage(metaDB mTypes.MetaDB, storeController storage.StoreController,
 func ParseRepo(repo string, metaDB mTypes.MetaDB, storeController storage.StoreController, log log.Logger) error {
 	imageStore := storeController.GetImageStore(repo)
 
+	var lockLatency time.Time
```

Comment:
why not lock the entire function? and not just parts, repo can be modified between reading its index.json here and checking its signatures leading to errors, right?

Reply:
If it's running before server, and before Task scheduler, then it's fine :D